resources/assets/components/utilities/NewsletterSubscription/SingleNewsletterSubscriptionForm.js

```diff
@@ -26,24 +33,44 @@ const SingleNewsletterSubscriptionForm = ({ emailSubscriptionTopic }) => {
   const handleOnFocus = event => {
     event.preventDefault();
 
-    // @TODO: add analytics tracking
-    console.log('handling input focus...');
+    trackAnalyticsEvent('focused_signup_newsletter', {
+      action: 'field_focused',
+      category: EVENT_CATEGORIES.siteAction,
+      label: 'signup_newsletter',
+      context: { contextSource: `newsletter_${emailSubscriptionTopic}` },
+    });
   };
 
   const handleSubmit = event => {
     event.preventDefault();
 
+    trackAnalyticsEvent('clicked_signup_newsletter', {
+      action: 'button_clicked',
+      category: EVENT_CATEGORIES.signup,
+      label: 'newsletter',
```

Comment:
Should this one also be consistent?

```suggestion
      label: 'signup_newsletter',
```